app/Community/Actions/ForwardMessageToDiscordAction.php

```diff
@@ -418,6 +439,51 @@ private function buildDiscordPayload(
         ];
     }
 
+    /**
+     * Forward a team account's reply to an existing Discord thread.
+     */
+    private function forwardTeamReplyToDiscord(
+        array $senderInboxConfig,
+        string $discordThreadId,
+        User $userFrom,
+        MessageThread $messageThread,
+        Message $message
+    ): void {
+        $webhookUrl = $senderInboxConfig['url'];
+        $fullBody = Shortcode::stripAndClamp($message->body, self::MESSAGE_BODY_MAX_LENGTH, preserveWhitespace: true);
+
+        if (empty($fullBody)) {
+            return;
+        }
+
+        // Post the reply to the existing Discord thread.
```

Comment:
Isn't the rest of this function just `sendMessagesToDiscardThread` where `$userTo` == `$userFrom`?